optuna_integration/sklearn/sklearn.py

```diff
@@ -636,17 +641,20 @@ def decision_function(
 
         return self.best_estimator_.decision_function(X, **kwargs)
 
-    @property
-    def inverse_transform(self) -> Callable[..., TwoDimArrayLikeType]:
+    def inverse_transform(
+        self, X: TwoDimArrayLikeType, *args: Any, **kwargs: Any
+    ) -> TwoDimArrayLikeType:
         """Call ``inverse_transform`` on the best estimator.
 
         This is available only if the underlying estimator supports
         ``inverse_transform`` and ``refit`` is set to :obj:`True`.
+        Note that we support the argument detailed in
+        https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.FunctionTransformer.html#sklearn.preprocessing.FunctionTransformer.inverse_transform
```

Comment:
I think we could simply remove these lines since all positional / keyword arguments are supported. What do you think?

```suggestion
```